scenes/lessons/7th_class/a_b+c+d/text.py

```diff
@@ -0,0 +1,9 @@
+from constants import LanguageConfig
+
+if LanguageConfig.is_armenian():
+    from .armenian_text import *
+else:
+    from .english_text import *
+
+__all__ = [
+]
```

Comment:
If you don't have any text, the video is both in Armenian and English :D You don't need to create those `text` files, just the scene file will do.